src/main/java/stsjorbsmod/cards/cull/ReapAndSow.java

```diff
@@ -0,0 +1,61 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.IncreaseManifestAction;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.powers.OverkillPower;
+import stsjorbsmod.powers.ReapAndSowPower;
+
+public class ReapAndSow extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(ReapAndSow.class);
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 2;
+    private static final int DAMAGE = 25;
+    private static final int UPGRADE_DAMAGE = 8;
+
+    private int overkill = 0;
+
+    public ReapAndSow() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = damage = DAMAGE;
+        magicNumber = baseMagicNumber = 0;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        overkill = getOverKill(m);
+        if (overkill > 0 && m.hasPower("Minion")) {
```

Comment:
This is missing a few edge cases (eg, enemies with half-dead states like awakened one and darklings). I think the best option would be to use `DamageWithOnKillEffectAction` instead of a plain `DamageAction` and move the `ReapAndSowPower` application to the on-kill callback passed to that action.